samri/pipelines/nipype_based/diagnostics.py

```diff
@@ -149,15 +150,25 @@ def diagnose(measurements_base,
 			(s_bru2nii, datasink, [('nii_file', 'anat')]),
 			])
 
+	#TODO: realigner  optional / als flag (wenn spm) ansonsten mandatory, slicetiming nicht spm
+	#TODO: realign vs. not realign, ICA geht verloren auf neurostars posten
 	if realign:
```

Comment:
Nested if statements are unhandy and here also clutter the function parameter space (I am very guilty of that myself :-/ ). In this case the conditional is better addressed with realign="one string or another" and elif statements.